### ISSUE TITLE
3.60+ mimetype detection (for some, like doc/docx)

### ISSUE THREAD

**Issue by sberthier** [seaweedfs/seaweedfs#5170]
**Describe the bug**

There is a regression from 3.59 to 3.60 with some (doc/docx at least) mimetype detection

When uploading a file without defined mimetype

```
echo "data" > data.tmp
curl -F file=@data.tmp "http://localhost:8888/data.jpg"
curl -F file=@data.tmp "http://localhost:8888/data.pdf"
curl -F file=@data.tmp "http://localhost:8888/data.doc"
curl -F file=@data.tmp "http://localhost:8888/data.docx"
```

- `curl -I "http://localhost:8888/data.jpg"` return "image/jpeg" in both version
- `curl -I "http://localhost:8888/data.pdf"` return "image/pdf" in both version

however

- `curl -I "http://localhost:8888/data.doc"` return **application/msword** in 3.59, but **application/octet-stream** in 3.60+
- `curl -I "http://localhost:8888/data.docx"` return **application/vnd.openxmlformats-officedocument.wordprocessingml.document** in 3.59, but **application/octet-stream** in 3.60+

**System Setup**
- `docker run -p 8888:8888 chrislusf/seaweedfs:3.61 server -master -master.defaultReplication=000 -volume -filer -ip=seaweedfs -ip.bind=0.0.0.0`

**Comment by chrislusf** [comment]
check the `/etc/mime.types` file on the linux.

**Comment by sberthier** [comment]
Yes, `/etc/mime.types` is the point for that kind of problem. On my side, I bypass it by just setting `content-type` during file upload.

As explain in [mime/type.go](https://cs.opensource.google/go/go/+/refs/tags/go1.21.5:src/mime/type.go;bpv=0), there are several possible file, but none of theme exists in both 3.59 and 3.60 seaweedfs docker image.
And there is effectively a small set of common builtin types (with *image/jpeg* and *application/pdf*).

The diff between version is the inclusion of **rclone**, that has been remove from standard build to [reduce bundle size](https://github.com/seaweedfs/seaweedfs/commit/7a50762b9ed05f1410a24f52571b49a25cd561eb) . There should be a dependency that add more mapping mime types, like [fs/mimetype.go](https://github.com/rclone/rclone/blob/master/fs/mimetype.go) but others too certainly.

Shouldn't be `rclone` added in  https://github.com/seaweedfs/seaweedfs/blob/master/.github/workflows/container_release4.yml `TAGS` since not in standard build anymore ?

Apart from that, for me it is ok; we can close this issue.
